spec/models/replication_transfer_spec.rb

```diff
@@ -44,7 +44,7 @@
     it "cancels bag_man_request when cancelled" do
       r = Fabricate(:replication_transfer, to_node: Fabricate(:local_node))
       allow(r.bag_man_request).to receive(:cancel!)
-      r.cancel!('other')
+      r.cancel!('other', 'detail')
       expect(r.bag_man_request).to have_received(:cancel!)
     end
```

Comment:
This spec also hides the circularity in the model calls for cancel.  If the `allow` on `r.bag_man_request.cancel!` is replaced with an `expect...to receive...`, then the last line can be deleted.  It might catch if there is more than one call to `r.bag_man_request.cancel!` but I don't think it will.  There might be more than one call to `r.cancel!`.